tools/release_provenance/models.py

```diff
@@ -1,20 +1,18 @@
+import os
 from sqlalchemy import (
     DateTime, Text, String, Column, ForeignKey )
 from sqlalchemy import create_engine
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import relationship, sessionmaker
 
-USERNAME = ""
-PASSWORD = ""
-HOST = "ceqv5w67lzc.db.cloud.edu.au"
-DATABASE = "TEST_RELEASE_PROVENANCE"
-CONNSTR = f'postgresql://{USERNAME}:{PASSWORD}@{HOST}/{DATABASE}'
-
 Base = declarative_base()
 
 def create_session():
+    connection = os.getenv("BUILD_DB_CONNECTION_STR")
```

Comment:
Could you just add a comment about what form the connection string should take, e.g. just a made-up example. Secrets aren't viewable, so if you needed to change it would be good to have some idea about what it should look like.

Reply:
Sure.